pageserver/src/tenant.rs

```diff
@@ -6785,10 +6785,14 @@ mod tests {
         for read in reads {
             info!("Doing vectored read on {:?}", read);
 
+            let query = VersionedKeySpaceQuery::Uniform {
+                keyspace: read.clone(),
+                lsn: reads_lsn,
+            };
```

Comment:
All these tests could be using the `VersionedKeySpaceQuery::uniform()` constructor